### ISSUE TITLE
Test suite started to fail (with newest Email::Mailer?)

### ISSUE THREAD

**Issue by eserte** [gryphonshafer/Log-Dispatch-Email-Mailer#3]
On my smoker systems:
```
#   Failed test 'html_attachment.qr'
#   at t/module.t line 172.
#                   'From: from@example.com
# Subject: Alert Log Message
# To: foo@example.com,bar@example.org
# Date: Tue, 8 Sep 2020 12:11:04 +0000
# MIME-Version: 1.0
# Content-Type: multipart/mixed; boundary=15995670641.FC4Fd00.2505
# Content-Transfer-Encoding: 7bit
# 
# 
# --15995670641.FC4Fd00.2505
# Date: Tue, 8 Sep 2020 12:11:04 +0000
# MIME-Version: 1.0
# Content-Type: multipart/alternative; boundary=15995670640.bacA7.2505
# Content-Transfer-Encoding: 7bit
# 
# 
# --15995670640.bacA7.2505
# Date: Tue, 8 Sep 2020 12:11:04 +0000
# MIME-Version: 1.0
# Content-Type: text/plain
# 
# This is a generic message: This is to alert you something happened..
# 
# --15995670640.bacA7.2505
# Date: Tue, 8 Sep 2020 12:11:04 +0000
# MIME-Version: 1.0
# Content-Type: text/html; charset=UTF-8
# Content-Transfer-Encoding: quoted-printable
# 
# <p>This is a generic message: <b>This is to alert you something happened.</=
# b>.</p>=
# 
# --15995670640.bacA7.2505--
# 
# --15995670641.FC4Fd00.2505
# Date: Tue, 8 Sep 2020 12:11:04 +0000
# MIME-Version: 1.0
# Content-Type: text/plain; name=log_file.txt
# Content-Disposition: attachment; filename=log_file.txt
# Content-Transfer-Encoding: base64
# 
# VGhpcyBpcyBwbGFpbiB0ZXh0IGF0dGFjaG1lbnQgY29udGVudC4=
# 
# --15995670641.FC4Fd00.2505--
# '
#     doesn't match '(?^ms:From: from@example.com\s+Subject: Alert Log Message\s+To: foo@example.com,bar@example.org\s+Date: [^\n]+\s+MIME-Version: 1\.0\s+Content-Type: multipart/mixed; boundary="[^"]+"\s+Content-Transfer-Encoding: 7bit\s+--\S+\s+Date: [^\n]+\s+MIME-Version: 1\.0\s+Content-Type: multipart/alternative; boundary="[^"]+"\s+Content-Transfer-Encoding: 7bit\s+--\S+\s+Date: [^\n]+\s+MIME-Version: 1\.0\s+Content-Type: text/plain\s+This is a generic message: This is to alert you something happened..\s+--\S+\s+Date: [^\n]+\s+MIME-Version: 1\.0\s+Content-Type: text/html; charset="UTF-8"\s+Content-Transfer-Encoding: quoted-printable\s+<p>This is a generic message: <b>This is to alert you something happened.</=\s+b>.</p>=\s+--[^\-]+--\s+--\S+\s+Date: [^\n]+\s+MIME-Version: 1\.0\s+Content-Type: text/plain; name="log_file.txt"\s+[^:]+: [^\n]+\s+[^:]+: [^\n]+\s+\S+=\s+--[^\-]+--)'
# Looks like you failed 1 test of 19.
t/module.t ............... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/19 subtests 
```

Statistical analysis suggests that this happens if Email::Mailer 1.11 is installed:
```
****************************************************************
Regression 'mod:Email::Mailer'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	38597405934694944.00
[1='eq_1.11']  	     -1.0000	      0.0000	-22459003601166872.00

R^2= 1.000, N= 127, K= 2
****************************************************************
```

**Comment by gryphonshafer** [comment]
Looks like an outdated regex for the test pattern to match. Resolved with a42a44b. Thanks for the report.

**Comment by eserte** [comment]
@gryphonshafer: I again see similar failures, but now with Email::Mailer 1.12. Statistical analysis:
```
****************************************************************
Regression 'mod:Email::Mailer'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	73042380741491952.00
[1='eq_1.10']  	      0.0000	      0.0000	   6.94
[2='eq_1.11']  	      0.0000	      0.0000	   7.89
[3='eq_1.12']  	     -1.0000	      0.0000	-63256557275027280.00

R^2= 1.000, N= 98, K= 4
****************************************************************
```

**Comment by gryphonshafer** [comment]
Oops. I didn't account for the change in headers from 1.12, but I've fixed that now with ce5190b. Thanks for the report.
